### ISSUE TITLE
Cannot find name 'HeadersInit'.

### ISSUE THREAD

**Issue by kms0219kms** [openapi-ts/openapi-typescript#1735]
**Description**

```
node_modules/.pnpm/openapi-fetch@0.10.2/node_modules/openapi-fetch/dist/index.d.ts:27:5 - error TS2304: Cannot find name 'HeadersInit'.

27   | HeadersInit
       ~~~~~~~~~~~


Found 1 error in node_modules/.pnpm/openapi-fetch@0.10.2/node_modules/openapi-fetch/dist/index.d.ts:27

 ELIFECYCLE  Command failed with exit code 1.
node:internal/modules/cjs/loader:1215
  throw err;
  ^
```

**Reproduction**

```
You can just write the code that simply request path using openapi-fetch, and try compile it using tsc to reproduction.
```

**Expected result**

```
Build successfully w/o any exceptions
```

**Checklist**

- [ ] I’m willing to open a PR (see [CONTRIBUTING.md](https://github.com/openapi-ts/openapi-typescript/blob/main/packages/openapi-fetch/CONTRIBUTING.md))

**Comment by phk422** [comment]
> **Description**
> 
> ```
> node_modules/.pnpm/openapi-fetch@0.10.2/node_modules/openapi-fetch/dist/index.d.ts:27:5 - error TS2304: Cannot find name 'HeadersInit'.
> 
> 27   | HeadersInit
>        ~~~~~~~~~~~
> 
> 
> Found 1 error in node_modules/.pnpm/openapi-fetch@0.10.2/node_modules/openapi-fetch/dist/index.d.ts:27
> 
>  ELIFECYCLE  Command failed with exit code 1.
> node:internal/modules/cjs/loader:1215
>   throw err;
>   ^
> ```
> 
> **Reproduction**
> 
> ```
> You can just write the code that simply request path using openapi-fetch, and try compile it using tsc to reproduction.
> ```
> 
> **Expected result**
> 
> ```
> Build successfully w/o any exceptions
> ```
> 
> **Checklist**
> 
> * [ ]  I’m willing to open a PR (see [CONTRIBUTING.md](https://github.com/openapi-ts/openapi-typescript/blob/main/packages/openapi-fetch/CONTRIBUTING.md))
I wrote a case, but this did not happen, can you provide a detailed case of repetition? And your version of node,typescript?

**Comment by kms0219kms** [comment]
[just-for-test.zip](https://github.com/user-attachments/files/16056177/just-for-test.zip)
This is simple reproduction for this issue. You may need to install pnpm@9.4.0 w/ corepack to check this reproduction.

**Comment by edonv** [comment]
+1

**Comment by edonv** [comment]
Any updates on this? I'm having the same issue.

**Comment by ngraef** [comment]
Fixed in `openapi-fetch@0.10.6`
